sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/LocalDocumentStorageTest.java

```diff
@@ -133,6 +133,18 @@ public void updateGetsCalledInWriteWithPendingOperation() {
         assertNotNull(argumentCaptor.getValue());
     }
 
+    @Test
+    public void readClosesDatabaseCursor() {
+        when(mDatabaseManager.getCursor(anyString(), any(SQLiteQueryBuilder.class), any(String[].class), any(String[].class), anyString())).thenReturn(mCursor);
+        when(mDatabaseManager.nextValues(mCursor)).thenReturn(null);
```

Comment:
```
        when(mDatabaseManager.nextValues(mCursor)).thenThrow(new SQLiteException());
```
or something like that, I removed suggestion as not sure if needs an import or exact syntax.
Also should reflect in title of method.

Testing with sqlite exception has better semantics as trigger a null pointer.

Reply:
After further inspection, I realized the way we access the cursor in this method can be refactored so that we can close the cursor without any try-finally. So no try-finally-with-closing-cursor is needed and we're closing cursor immediately after the usage.